ADR/rollup.md

```diff
@@ -0,0 +1,33 @@
+# Rollup
+
+version: 2.67.3
+Date: 2022-02-02
+
+### Features
+
+- Improve side effect detection when using Array.prototype.groupBy/groupByToMap (#4360)
```

Comment:
I think we are using the ADR here to keep a copy of the changelog of a library itself — I would prefer if we keep our ADRs only for those changes, discussions or problems that affect our internal code. 
For instance if there's a library update that introduces API changes that make our code incompatible, that's the kind of information we would want to keep in the ADR.